examples/Hazard3/Hazard3.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/sh
+
+# abort on error
+set -e
+
+# clone Hazard3 repo if not done yet
+if [ ! -e Hazard3/.git ] ; then
+  git clone https://github.com/Wren6991/Hazard3
+  (cd Hazard3 ; git checkout v1.0)
+fi
+
+cd Hazard3
+
+ebmc -I hdl \
+  -D HAZARD3_ASSERTIONS --bound 0 --top hazard3_alu \
+  hdl/arith/hazard3_alu.v \
+  hdl/arith/hazard3_priority_encode.v \
+  hdl/arith/hazard3_onehot_encode.v \
+  hdl/arith/hazard3_onehot_priority.v \
+  hdl/arith/hazard3_shift_barrel.v
+
+# expected elaboration-time constant, but got `hazard3_muldiv_seq.properties.i'
+# $past, for loop
+# ebmc -I hdl -D HAZARD3_ASSERTIONS --systemverilog --bound 0 hdl/arith/hazard3_muldiv_seq.v
+
+ebmc -I hdl \
+  -D HAZARD3_ASSERTIONS --bound 0 \
+  hdl/arith/hazard3_shift_barrel.v
+
+# conflicting assignment types
```

Comment:
Here and below: do those persist? If so, can we have KNOWNBUG tests for them?

Reply:
Yes; one test added, the other one linked as a comment